aspnetcore/mvc/views/tag-helpers/built-in/distributed-cache-tag-helper.md

```diff
@@ -3,38 +3,48 @@ title: Distributed Cache Tag Helper in ASP.NET Core
 author: pkellner
 description: Learn how to use the Distributed Cache Tag Helper.
 ms.author: riande
-ms.date: 02/14/2017
+ms.custom: mvc
+ms.date: 10/10/2018
 uid: mvc/views/tag-helpers/builtin-th/distributed-cache-tag-helper
 ---
 # Distributed Cache Tag Helper in ASP.NET Core
 
-By [Peter Kellner](http://peterkellner.net) 
+By [Peter Kellner](http://peterkellner.net)
 
 The Distributed Cache Tag Helper provides the ability to dramatically improve the performance of your ASP.NET Core app by caching its content to a distributed cache source.
 
-The Distributed Cache Tag Helper inherits from the same base class as the Cache Tag Helper. All attributes associated with the Cache Tag Helper will also work on the Distributed Tag Helper.
+For an overview of Tag Helpers, see <xref:mvc/views/tag-helpers/intro>.
 
-The Distributed Cache Tag Helper follows the **Explicit Dependencies Principle** known as **Constructor Injection**. Specifically, the `IDistributedCache` interface container is passed into the Distributed Cache Tag Helper's constructor. If no specific concrete implementation of `IDistributedCache` has been created in `ConfigureServices`, usually found in startup.cs, then the Distributed Cache Tag Helper will use the same in-memory provider for storing cached data as the basic Cache Tag Helper.
+The Distributed Cache Tag Helper inherits from the same base class as the Cache Tag Helper. All attributes associated with the Cache Tag Helper also work on the Distributed Tag Helper.
```

Comment:
All the Cache Tag Helper attributes are available to the Distributed Tag Helper.
or something like that
~also~